app/controllers/workflows_controller.rb

```diff
@@ -49,4 +52,7 @@ def turn_around_days
     params[:turn_around_days].to_i if params[:turn_around_days].present?
   end
 
+  def initial_state_name
+    params[:initial_state_name] || initial_state(params[:qc_flow], params[:cherrypick_flow])
+  end
```

Comment:
params[:cherrypick_flow] has inconsistent naming, snake_case here, camelCase above. (Snake case would be more typical , although I noticed hasComment is already camelCase for some reason)

Reply:
Hmm, your front end is sending camelCase, so I think this might result in setting an incorrect initial state, as it will always map to initial_state(false, false).

Reply:
Ah yes! That's a result of a later change in the front-end. I'm fixing it.

Reply:
Done in 452c548